v21.2/aws-dms.md

```diff
@@ -24,6 +24,9 @@ Complete the following items before starting this tutorial:
 - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.Creating.html) in AWS.
 - Configure a [source endpoint](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html) in AWS pointing to your source database.
 - Ensure you have a secure, publicly available CockroachDB cluster running v21.2.16 or later.
+- If you intend to migrate to a {{ site.data.products.db }} cluster, you must first **disable revision history** for cluster backups.
```

Comment:
Optional: Might be helpful to include a link to this page take-backups-with-revision-history-and-restore-from-a-point-in-time.html